data/changelog.txt

```diff
@@ -186,6 +212,14 @@ Limit the times assassins can try to kill you.
 Increase plastic variety, adjust plastics to be more realistic in terms of protection
 Characters start with basic skills from their previous life
 Climbing down stepladders is now safe; climbing down ledges tells you how risky it is.
+Adds Karate manual to Novice Martial Artist starting items
+Add NO_SPELLCASTING flag to Stunned effect
+Bulk loading/unloading  items into container saves time cost
+Improve pets' ability to use stairs
+Tweaked Mozzarella pricing
+Make ferals actually feel like human enemies instead of weaker zombies with range attacks
+Give long stick reach
+Most weapon techniques require some melee skill now
```

Comment:
```suggestion
Add NO_SPELLCASTING flag to Stunned effect
Improve pets' ability to use stairs
Make ferals actually feel like human enemies instead of weaker zombies with range attacks
```